react-main/first-app/src/app/components/App/Game/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+
+import Board from '../Board';
+
+import './styles.css';
+
+class Game extends Component {
+  state = { history: [{ squares: Array(9).fill(null)}], xIsNext: true };
+
+  handleClick(i) {
+    const history = this.state.history;
+    const current = history[history.length - 1];
+    const squares = current.squares.slice();
+    if (calculateWinner(squares) || squares[i]) {
+      return;
+    }
+    squares[i] = this.state.xIsNext ? 'X' : 'O';
+    this.setState({
+      history: history.concat([{
+          squares
+        }
+      ]),
+      xIsNext: !this.state.xIsNext
+    });
+  }
+  render() {
+    const history = this.state.history;
+    const current = history[history.length - 1];
+    const winner = calculateWinner(current.squares);
+
+    const moves = history.map((step, move) => {
+      const desc = move ?
+        'Go to move #' + move :
+        'Go to game start';
+      return (
+        <li>
+          <button onClick={() => this.jumpTo(move)}>{desc}</button>
+        </li>
+      );
+    });
+
+    let status;
```

Comment:
I would define status as ``const status = winner ? `Winner: ${winner}` : `Next player: ${this.state.xIsNext ? 'X' : 'O'}``